enterprise/frontend/src/embedding-sdk/cli/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-literal-metabase-strings": "off",
```

Comment:
ESLint is hierachical, so we can create a `.eslintrc` that turns off "no-literal-metabase-strings" and "no-console" for the CLI, since we are in a CLI.